src/NuGet.Clients/PackageManagement.VisualStudio/Utility/EnvDTEProjectUtility.cs

```diff
@@ -1355,8 +1355,15 @@ internal static void Save(EnvDTEProject project)
         {
             ThreadHelper.ThrowIfNotOnUIThread();
 
-            FileSystemUtility.MakeWritable(project.FullName);
-            project.Save();
+            try
+            {
+                FileSystemUtility.MakeWritable(project.FullName);
+                project.Save();
+            }
+            catch (Exception ex)
```

Comment:
Why are we not sharing these to the user log?

What exceptions are being thrown here? And why don't we retry?